utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -330,9 +330,29 @@ function Set-TemplateReferencesSection {
     $TextInfo = (Get-Culture).TextInfo
     foreach ($resourceType in $relevantResourceTypes) {
         $Type, $Resource = $resourceType.Type -split '/', 2
```

Comment:
I know that was my code, but still: Can we rename that to `$provider, $resourceType = `?

Reply:
Updated to use `$ProviderNamespace` as its more aligned to what we see in the docs, and `$ResourceType`, had to update other things as well like the foreach and `SectionContent`, Please check.